packages/documentation-site/patternfly-docs/content/design-guidelines/components/alert/alert.md

```diff
@@ -89,11 +126,29 @@ Use bordered inline alerts to return feedback as the result of a user action wit
  
   <img src="./img/Inline-alert-specific-context.png" alt="inline alert inline with content" />
 
-* **If the alert is being used to indicate errors,** use an error validation summary at the top of the page to indicate what errors are present. The summary should be a quick, general overview. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fit it. 
+* **If the alert is being used to indicate errors,** use an error validation summary at the top of the page to indicate what errors are present. The summary should be a quick, general overview. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fix it. 
+
+  <img src="./img/inline-alert-page-level-with-link-one.png" alt="inline alert inline with link demo 1" /> 
+
+  <img src="./img/inline-alert-page-level-with-link-two.png" alt="inline alert inline with link demo 2" />
+
+* **If multiple errors are present in the same location,** order them from most to least severe. 
+
+  <img src="./img/Multiplealert.png" alt="multiple alerts on a page" />
+
+* **If the alert is being used to indicate errors on a modal,** use an error validation summary at the top of the modal below the title to indicate what errors are present. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fix it. 
+
+  <img src="./img/inline-alert-modal-with-link-one.png" alt="multiple alerts on a modal demo 1" /> 
```

Comment:
I would edit this picture. It doesn't look like a modal.. I would make it so that the modal is above a grey background that covers what's behind the modal. The way it looks in the real UI.

Reply:
Or maybe just delete this picture, and only keep demo 2 of the modal

Reply:
Deleting the demo 1 image is a better option.

Reply:
@pankajshivpuje ok then whichever you think makes more sense!